http-clients/src/main/java/com/palantir/remoting/http/GuavaOptionalAwareContract.java

```diff
@@ -67,7 +68,7 @@ public GuavaOptionalAwareContract(Contract delegate) {
                     Class<?> cls = parameterTypes[i];
                     if (cls.equals(Optional.class)) {
                         FluentIterable<Class<?>> paramAnnotations =
-                                FluentIterable.of(annotations[i]).transform(EXTRACT_CLASS);
+                                FluentIterable.from(Lists.newArrayList(annotations[i])).transform(EXTRACT_CLASS);
```

Comment:
went with inlining exactly what was in guava

  public static <E> FluentIterable<E> of(E[] elements) {
    return from(Lists.newArrayList(elements));
  }